src/Providers/DataGrid/Wijmo/Language/Languages.ts

```diff
@@ -13,6 +13,7 @@ namespace Providers.DataGrid.Wijmo.Language {
             case 'en-CA':
             case 'en-GB':
             case 'es-MX':
+            case 'fr-CA':
```

Comment:
@OS-giulianasilva We can add this to the release notes if it wasn't supported before

Reply:
done